tests/e2e/specs/backend/mini-cart.test.js

```diff
@@ -37,6 +49,16 @@ const addBlockToWidgetsArea = async () => {
 	await miniCartButton[ 0 ].click();
 };
 
+// insertBlock gets focus on the canvas, so the compatibility notices popup doesn't appear.
+// I created this function to avoid that.
+const addBlockToFSEArea = async () => {
+	await searchForBlockFSE( block.name );
+	const insertButton = await page.waitForXPath(
+		`//button//span[contains(text(), '${ block.name }')]`
+	);
+	await insertButton.click();
+};
+
```

Comment:
Since this utility function is not specific to the `mini-cart` but could be useful for other tests, what about migrating it in the `utils` file and making it generic?